OpenRA.Game/Translation.cs

```diff
@@ -88,8 +92,8 @@ public string GetString(string key, IDictionary<string, object> arguments = null
 
 		public bool TryGetString(string key, out string value, IDictionary<string, object> arguments = null)
 		{
-			if (string.IsNullOrEmpty(key))
-				throw new ArgumentException("A translation key must not be null or empty.", nameof(key));
+			if (key == null)
+				throw new ArgumentException("A translation key must not be null.", nameof(key));
```

Comment:
`ArgumentNullException` since it's now a strict null check. (I also don't often bother with a custom message for `ArgumentNullException` since there's usually no value to add, but not too fussed on that one)